src/components/Footer.tsx

```diff
@@ -0,0 +1,171 @@
+import Container from './Container'
+import { Logo } from './Logo'
+
+interface MenuItemsProps {
+  path: string
+  title: string
+}
+
+const MenuItems = ({ path, title }: MenuItemsProps) => {
+  return (
+    <a
+      href={`#${path}`}
+      aria-current="page"
+      
+    >
+      {title}
+    </a>
+  )
+}
+
+
+export const Footer = ({ navbar, logo }) => {
+    const { items} = navbar
+
+  return (
+    <Container>
+      <footer className="bg-white bottom-0">
+        <hr className="my-2 border-gray-100 sm:mx-auto lg:my-8" />
+        <div className="mx-auto w-full p-4 py-2 lg:py-8">
+          <div className="md:flex md:justify-between">
+            <div className="mb-4 flex items-start space-x-2 rtl:space-x-reverse">
+              <Logo logo={logo} />
+            </div>
+            <div className="grid grid-cols-2 gap-8 sm:gap-6 sm:grid-cols-4">
+              {items.length > 0 && (
+                <div>
+                  <h2 className="mb-3 font-bold">Categories</h2>
+                  <ul>
+                    {items.map((menuItem) => (
+                      <li key={menuItem.id} className="mb-2 hover:underline">
+                        <MenuItems
+                          path={menuItem.path}
+                          title={menuItem.title}
+                        />
+                      </li>
+                    ))}
+                  </ul>
+                </div>
+              )}
+              <div>
+                <h2 className="mb-3 font-bold">Regulations</h2>
+                <ul>
+                  <li className="mb-2">
+                    <a href="#" className="hover:underline">
+                      Privacy Policy
+                    </a>
+                  </li>
+                  <li className="mb-2">
+                    <a href="#" className="hover:underline">
+                      Terms &amp; Conditions
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" className="hover:underline">
+                      Cookie settings
+                    </a>
+                  </li>
+                </ul>
+              </div>
+              <div>
+                <h2 className="mb-3 font-bold">Contacts</h2>
+                <ul>
+                  <li className="mb-2">
+                    <a
+                      href="mailto:someone@example.com"
+                      className="hover:underline "
+                    >
+                      Email
+                    </a>
+                  </li>
+                  <li>
+                    <a href="" className="hover:underline">
+                      LinkedIn
+                    </a>
+                  </li>
+                </ul>
+              </div>
+              <div className="invisible lg:visible">
+                {/* We can add search later */}
```

Comment:
I would remove Search, I have no idea when it will be implemented and if we even need it